UWGeodynamics/_model.py

```diff
@@ -164,6 +164,12 @@ def __init__(self, elementRes=(64, 64),
                                      minCoord=minCoord,
                                      maxCoord=maxCoord,
                                      periodic=self.periodic)
+                                     
+        self.init_mesh = FeMesh_Cartesian(elementType=self.elementType,
+                                     elementRes=self.elementRes,
+                                     minCoord=minCoord,
+                                     maxCoord=maxCoord,
+                                     periodic=self.periodic)
```

Comment:
Please Don't add the init_mesh in the Model. It should be in the FreeSurface

Reply:
Thanks, Romain, 
the init_mesh was changed into the _freesurface.py. see the new commit:
https://github.com/underworldcode/UWGeodynamics/pull/250/commits/43374a2c0bee06987784f34a0af29b157d763672